CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# How to contribute
+
+## Submitting features or changes
+
+If you've been added as a collaborator, then clone project to your local machine:
+
+```
+git clone https://github.com/itsevalieu/planetb
+```
+
+Otherwise, fork the project (https://github.com/itsevalieu/planetb/fork), then clone the forked project to your local machine:
+
+```
+git clone https://github.com/<yourusername>/planetb
+```
+
+Checkout `develop` branch to create a new feature branch. Use the branch naming scheme, `issue<issue no>-feature-title`:
+
+```
+git checkout develop
+git checkout -b issue42-add-logarithms
+```
+
+Use a present tense commit message, preferably one that's no more than 50 characters in length:
+
+```
+git commit -am "add logarithms"
+```
```

Comment:
Are we starting commit messages with capitalization or does that not matter? ie: "Add logarithms" vs "add logarithms."

Reply:
I'm thinking we should reserve capitalized commit messages for "_important_" commits - emphasis on the subjectivity of the word "_important_".

Reply:
Sounds goood! Maybe should note that.

Reply:
added some notes